schematic/help.py

```diff
@@ -93,3 +93,21 @@
         }
     }
 }
+
+
+# `schematic init` command description
+init_command = {
+    "init": {
+        "short_help": (
+            "Initialize mode of authentication for schematic."
+        ),
+        "auth": (
+            "Specify the mode of authentication you want to use. You can use one of "
```

Comment:
Can we specify which service the authentication is for (the user could be confused whether this is auth for Google Sheet API, synapse, or something else).